app/src/main/res/navigation/mobile_navigation.xml

```diff
@@ -2,26 +2,8 @@
 <navigation xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/mobile_navigation"
-    app:startDestination="@+id/nav_main">
```

Comment:
This is apparently not a big deal, but you deleted app:startDestination without replacing it. Is it OK? Should we not put `app:startDestination="@+id/nav_profile"`?